### ISSUE TITLE
Release `3.24.0` can't be linked

### ISSUE THREAD

**Issue by mathaeus** [Appboy/appboy-ios-sdk#233]
Hi, I wanted to update to release `3.24.0` this morning, but I get linking errors (at least for the simulator, didn't try building for release)
`3.23.0` works just fine.

I'm integrating the SDK via `cocoapods`.

Do you have any idea what's going on there?
Thanks

```
ld: warning: ignoring file /.../Appboy-iOS-SDK/AppboyKit/libAppboyKitLibrary.a, building for iOS Simulator-x86_64 but attempting to link with file built for unknown-unsupported file format ( 0x76 0x65 0x72 0x73 0x69 0x6F 0x6E 0x20 0x68 0x74 0x74 0x70 0x73 0x3A 0x2F 0x2F )
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_ABKInAppMessageHTML", referenced from:
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageUIController.o)
     (maybe you meant: _OBJC_CLASS_$_ABKInAppMessageHTMLViewController, _OBJC_CLASS_$_ABKInAppMessageHTMLBaseViewController , _OBJC_CLASS_$_ABKInAppMessageHTMLFullViewController )
  "_OBJC_CLASS_$_ABKInAppMessageHTMLFull", referenced from:
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageUIController.o)
     (maybe you meant: _OBJC_CLASS_$_ABKInAppMessageHTMLFullViewController)
  "_OBJC_CLASS_$_ABKInAppMessageModal", referenced from:
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageUIController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageImmersiveViewController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageWindowController.o)
     (maybe you meant: _OBJC_CLASS_$_ABKInAppMessageModalViewController)
  "_OBJC_CLASS_$_ABKInAppMessageSlideup", referenced from:
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageUIController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageSlideupViewController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageWindowController.o)
     (maybe you meant: _OBJC_CLASS_$_ABKInAppMessageSlideupViewController)
  "_OBJC_CLASS_$_ABKPushUtils", referenced from:
      objc-class-ref in BrazeManager.o
  "_OBJC_CLASS_$_ABKInAppMessageWebViewBridge", referenced from:
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageHTMLBaseViewController.o)
  "_ABKURLDelegateKey", referenced from:
      Freeletics.BrazeManager.(brazeOptions in _FF63281FC8A3E3809A21C5C32DD4DD75)() -> [Swift.String : Any] in BrazeManager.o
  "_ABKContentCardsProcessedNotification", referenced from:
      Freeletics.FreeTodayContentCardDataProvider.(registerContentCardUpdateObserver in _B6C70F236E51F693CD4D42109FD4E43E)() -> () in FreeTodayContentCardViewModelDataProvider.o
      -[ABKContentCardsTableViewController setUp] in Appboy_iOS_SDK(ABKContentCardsTableViewController.o)
  "_OBJC_CLASS_$_ABKInAppMessageImmersive", referenced from:
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageModalViewController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageFullViewController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageUIController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageImmersiveViewController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageWindowController.o)
     (maybe you meant: _OBJC_CLASS_$_ABKInAppMessageImmersiveViewController)
  "_OBJC_CLASS_$_ABKClassicContentCard", referenced from:
      objc-class-ref in FreeTodayContentCardViewModel.o
      objc-class-ref in Appboy_iOS_SDK(ABKClassicImageContentCardCell.o)
      objc-class-ref in Appboy_iOS_SDK(ABKContentCardsTableViewController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKClassicContentCardCell.o)
     (maybe you meant: _OBJC_CLASS_$_ABKClassicContentCardCell)
  "_OBJC_CLASS_$_ABKContentCard", referenced from:
      objc-class-ref in FreeTodayContentCardViewModelDataProvider.o
     (maybe you meant: _OBJC_CLASS_$_ABKContentCardsTableViewController, _OBJC_CLASS_$_ABKContentCardsWebViewController , _OBJC_CLASS_$_ABKContentCardsViewController )
  "_OBJC_CLASS_$_ABKInAppMessageFull", referenced from:
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageUIController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageImmersiveViewController.o)
     (maybe you meant: _OBJC_CLASS_$_ABKInAppMessageFullViewController)
  "_OBJC_CLASS_$_ABKCaptionedImageContentCard", referenced from:
      objc-class-ref in FreeTodayContentCardViewModel.o
      objc-class-ref in Appboy_iOS_SDK(ABKCaptionedImageContentCardCell.o)
      objc-class-ref in Appboy_iOS_SDK(ABKContentCardsTableViewController.o)
     (maybe you meant: _OBJC_CLASS_$_ABKCaptionedImageContentCardCell)
  "_OBJC_CLASS_$_Appboy", referenced from:
      objc-class-ref in BrazeManager.o
      objc-class-ref in FreeTodayContentCardViewModelDataProvider.o
      objc-class-ref in Tracking+BrazeService.o
      objc-class-ref in Appboy_iOS_SDK(ABKContentCardsTableViewController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageHTMLBaseViewController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageUIController.o)
      objc-class-ref in Appboy_iOS_SDK(ABKInAppMessageWindowController.o)
      ...
  "_ABKContentCardsProcessedIsSuccessfulKey", referenced from:
      closure #1 (Foundation.Notification) -> () in Freeletics.FreeTodayContentCardDataProvider.(registerContentCardUpdateObserver in _B6C70F236E51F693CD4D42109FD4E43E)() -> () in FreeTodayContentCardViewModelDataProvider.o
      -[ABKContentCardsTableViewController contentCardsUpdated:] in Appboy_iOS_SDK(ABKContentCardsTableViewController.o)
  "_OBJC_CLASS_$_ABKBannerContentCard", referenced from:
      objc-class-ref in FreeTodayContentCardViewModel.o
      objc-class-ref in Appboy_iOS_SDK(ABKBannerContentCardCell.o)
      objc-class-ref in Appboy_iOS_SDK(ABKContentCardsTableViewController.o)
     (maybe you meant: _OBJC_CLASS_$_ABKBannerContentCardCell)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by Palbrevik** [comment]
+1

**Comment by mathaeus** [comment]
I'm also `Xcode 11.5.0`.

**Comment by JamesLarcombe51** [comment]
I also had this issue with my circleci build machine. Forcing pod to use 3.23.0 fixed the issue

**Comment by hokstuff** [comment]
Hi @mathaeus,

Thanks for filing the issue, we are currently investigating to determine a fix for this.

**Comment by hokstuff** [comment]
Hi @mathaeus @Palbrevik @JamesLarcombe51,

We have just released a [patch version 3.24.1](https://github.com/Appboy/appboy-ios-sdk/releases/tag/3.24.1) that will handle the underlying issue with our Cocoapods integration. You should now be able to run `pod update` or `pod install` to download the latest pod. Let us know if you run into any issues.

Thanks!

**Comment by mathaeus** [comment]
Hi @hokstuff, looks good now. Thanks for the quick fix!
